election.py

```diff
@@ -205,44 +156,24 @@ class VoteTracker:
     """
 
     def __init__(self, votes_cast=0.0, votes_for_candidate=None):
-        """Initializes VoteTracker with votes cast and votes for candidates.
-
-        Args:
-            votes_cast: Float value of the total votes cast.
-            votes_for_candidate: Dict mapping Candidates to float values of
-                votes.
-        """
         self.votes_cast = votes_cast
         self._votes_for_candidate = (votes_for_candidate
                                      if votes_for_candidate is not None else dict())
 
     def __eq__(self, other):
-        """Checks equality between two VoteTrackers.
-
-        Args:
-            other: VoteTracker to check equality with.
-
-        Returns:
-            Boolean indicating if the VoteTrackers are equal or not.
-        """
         if isinstance(other, VoteTracker):
             return (self.votes_cast == other.votes_cast and
                     self._votes_for_candidate == other._votes_for_candidate)
 
     def __repr__(self):
-        """Returns a printable system representation of the VoteTracker.
-
-        Returns:
-            String containing the printable representation of the VoteTracker.
-        """
         return 'VoteTracker(votes_for_candidate={!r}, votes_cast={!r})'.format(
                 self._votes_for_candidate, self.votes_cast)
 
     def decription(self):
         """Returns a printable long-form user representation of the VoteTracker.
 
-        Returns:
-            String containing the printable representation of the VoteTracker.
+        :return String containing the printable representation of the
```

Comment:
There is a standard for autodocumentation:
https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html